src/pages/LearnPage/sampleLesson.ts

```diff
@@ -0,0 +1,160 @@
+        date: '04/13/2022'
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<p>This document will cover all required material for Exam 1. Good Luck!</p>",
+        } 
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<h2>Lists, Stacks and Queues</h2>",
+        } 
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<p>Lists, stacks, and queues are basic data structures meant to hold sequential data in an organized fashion.</p>",
+        } 
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<h3>Stacks</h3>",
+        } 
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<p>A stack is a linear data structure in which elements can be inserted and deleted only from one side of the list, called the <b>top</b>. A stack follows the <b>LIFO</b> (Last In First Out) principle, i.e., the element inserted at the last is the first element to come out. The insertion of an element into stack is called push operation, and deletion of an element from the stack is called <b>pop</b> operation. In stack we always keep track of the last element present in the list with a pointer called <b>top</b>.</p>",
+        } 
+    },
+    {
+        type: 'image',
+        content: {
+            html: '<img src="https://www.jquery-az.com/html/images/banana.jpg" title="Title of image" alt="alt text here"/>',
+            sourcePath: 'https://media.geeksforgeeks.org/wp-content/uploads/geek-stack-1.png',
+            title: 'Stack Data Structure',
+            caption: 'An operation with a stack data structure.',
+            size: 'md',
+            alignment: 'none'
+        }
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<h3>Queues</h3>",
+        } 
+    },
+    {
+        type: 'text',
+        content: {
+            html: "<p>A queue is a linear data structure in which elements can be inserted only from one side of the list called <b>rear</b>, and the elements can be deleted only from the other side called the <b>front</b>. The queue data structure follows the <b>FIFO</b> (First In First Out) principle, i.e. the element inserted at first in the list, is the first element to be removed from the list. The insertion of an element in a queue is called an <b>enqueue</b> operation and the deletion of an element is called a <b>dequeue</b> operation. In queue we always maintain two pointers, one pointing to the element which was inserted at the first and still present in the list with the <b>front</b> pointer and the second pointer pointing to the element inserted at the last with the <b>rear</b> pointer.</p>",
+        } 
+    },
+    {
+        type: 'image',
+        content: {
+            html: '<img src="https://www.jquery-az.com/html/images/banana.jpg" title="Title of image" alt="alt text here"/>',
+            sourcePath: 'https://media.geeksforgeeks.org/wp-content/uploads/geek-queue-1.png',
```

Comment:
Will the images be uploaded to our database in the long run? - Something to think about as we hook it up to Mongodb.

Reply:
Yes, will have to look into how images are stored with coding problems and apply the same process for lessons

Reply:
coding problem images are not stored in the database but are rendered after hosting them on a third party.